index.html

```diff
@@ -1629,7 +1629,8 @@ <h4>
         <a>recordType</a> attribute.
       </li>
       <li>
-        If the |recordType| value is "`smart-poster`", or an [=external type name=],
+        If the |recordType| value is "`smart-poster`", or if running
+        <a>validate external type</a> on |recordType| returns other than `false`,
```

Comment:
I hope you hate this sentence, too :)

Reply:
Other that false is ugly... just make the algo return true/false